alerter.cpp

```diff
@@ -1,14 +1,19 @@
 #include <iostream>
 #include <assert.h>
 
+#define THRESHOLD 250
+

```

Comment:
stub/test code and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.

The production code should not be aware of test code or test stub. It should be abstracted by the use of interface or function pointer.